modules/develop/pages/consume-data/consumer-offsets.adoc

```diff
@@ -4,20 +4,125 @@
+Within a partition, each message (once accepted and acknowledged by the partition leader) is permanently assigned a unique sequence number called an glossterm:offset[]. Offsets enable consumers to resume processing from a specific point, such as after an application outage. If an outage prevents your application from receiving events, you can use the consumer offset to retrieve only the events that occurred during the downtime. By default, the first message in a partition is assigned offset 0, the next is offset 1, and so on. You can manually specify a specific start value for offsets if needed. Once assigned, offsets are immutable, ensuring that the order of messages within a partition is preserved.
 
-NOTE: A consumer group is a set of consumers that cooperate to consume data from some topics. The partitions of all the topics are divided among the consumers in the group. As new group members arrive and old members leave, the partitions are re-assigned so that each member receives a proportional share of the partitions.
+As a consumer reads messages from Redpanda, it can save its progress by “committing the offset” (known as an glossterm:offset commit[]), an action initiated by the consumer, not by Redpanda. Kafka client libraries provide an API for committing offsets, which communicates with Redpanda using the glossterm:consumer group[] API. Each committed offset is stored as a message in the `pass:[__consumer_offsets]` topic, which is a private Redpanda broker topic that stores committed offsets from each Kafka consumer attached to Redpanda, allowing the consumer to resume processing from the last committed point. Note that Redpanda exposes the `pass:[__consumer_offsets]` key to enable the many tools in the Kafka ecosystem that rely on this value for their operation, providing greater ecosystem interoperability with environments and applications.
 
-When the group coordinator receives an OffsetCommitRequest, it appends the request to a special https://kafka.apache.org/documentation/#compaction[compacted^] Kafka topic named `pass:[__consumer_offsets]`.
+When a glossterm:consumer group[]) works together to consume data from topics, the partitions are divided among the consumers in the group. For example, if a topic has 12 partitions, and there are 2 consumers, each consumer would be assigned 6 partitions to consume. If a new consumer starts later and joins this consumer group, a rebalance occurs, such that each consumer ends up with 4 partitions to consume. You specify a consumer group by setting the `group.id` configuration parameter to a unique name for the group.
```

Comment:
```suggestion
When a consumer group works together to consume data from topics, the partitions are divided among the consumers in the group. For example, if a topic has 12 partitions, and there are 2 consumers, each consumer would be assigned 6 partitions to consume. If a new consumer starts later and joins this consumer group, a rebalance occurs, such that each consumer ends up with 4 partitions to consume. You specify a consumer group by setting the `group.id` configuration parameter to a unique name for the group.
```
Suggest removing this glossterm, since it's defined in the previous paragraph. But if you keep it, there's a typo in it that needs fixing!